CAFAna/Core/SpectrumLoader.cxx

```diff
@@ -348,6 +413,10 @@ void SpectrumLoader::HandleFile(TFile *f, Progress *prog) {
       }
     }
 
+#ifdef USE_TH2JAGGED
+    sr.OffAxisFluxConfig = -1;
+    sr.OffAxisFluxBin = -1;
+#endif
```

Comment:
I forget what the story was with TH2Jagged? I thought we could do without it?

Reply:
We will be able to do without it at some point in the near future. We are essentially waiting for a full set of new flux uncertainties from the beam group. Until then we are a bit stuck using this custom root object to read the off-axis flux uncertainties unless we want to pile a big job on to Luke. I have already mostly written the interface to read in a the new off-axis flux uncertainties in a way that does not need TH2Jagged, so when the systematics are ready, we should be able to just plug them in.